src/cloud-sql-instance.ts

```diff
@@ -42,6 +55,7 @@ interface CloudSQLInstanceOptions {
   ipType: IpAddressTypes;
   limitRateInterval?: number;
   sqlAdminFetcher: Fetcher;
+  checkDomainInterval?: number;
```

Comment:
should we keep this inline with other Connectors? (i.e. call it `failoverPeriod`)

Reply:
Renamed all occurrences  "failoverPeriod"